config/shared.lua

```diff
@@ -1,6 +1,7 @@
 return {
     useTarget = false, -- Enables ox_target interactions
     debugPoly = false,
+    discordWebhook = nil, -- Replace nil with your webhook if you chose to use discord logging over ox_lib logging
```

Comment:
should be a server config only, no? I guess it's fine if you want to keep all config in shared. I see that the target stuff is here as well which would be client only